syncd/syncd.cpp

```diff
@@ -2365,8 +2365,14 @@ void processFlexCounterEvent(
     const auto &key = kfvKey(kco);
     const auto &op = kfvOp(kco);
 
+    std::size_t delimiter = key.find_last_of(":");
+    const auto intervalStr = key.substr(delimiter + 1);
+    const auto vidStr = key.substr(0, delimiter);
+
+    int pollInterval = std::stoi(intervalStr);
```

Comment:
>pollInterval [](start = 8, length = 12)

I am not aware of the design discussion. Is there any used cases for multiple intervals in one switch? If yes, what is the usage for both set and remove?

Reply:
Port or queue could be removed as part of breakout process.